README.md

```diff
@@ -168,3 +168,44 @@ The container exposes /config as a volume. This is the directory where the suppl
 If you have transmission authentication enabled and want scripts in another container to access and
 control the transmission-daemon, this can be a handy way to access the credentials.
 For example, another container may pause or restrict transmission speeds while the server is streaming video.
+
+## Make it work an a Synology NAS
+Here are the steps to run it on a Synology NAS (Tested on DSM 6) :
+
+- Connect as _admin_ to your Synology SSH
+- Switch to root with command `sudo su -`
+- Enter your _admin_ password when prompted
+- Create a TUN.sh file anywhere in your synology file system by typing `vim /volume1/foldername/TUN.sh`
+replacing _foldername_ with any folder you created on your Synology
+- Paste @timkelty 's script :
+```
+#!/bin/sh
+
+# Create the necessary file structure for /dev/net/tun
+if ( [ ! -c /dev/net/tun ] ); then
+	if ( [ ! -d /dev/net ] ); then
+		mkdir -m 755 /dev/net
+	fi
+	mknod /dev/net/tun c 10 200
+fi
+
+# Load the tun module if not already loaded
+if ( !(lsmod | grep -q "^tun\s") ); then
+	insmod /lib/modules/tun.ko
+fi
+```
+- Save the file with [escape] + `:wq!`
+- Go in the folder containing your script : `cd /volume1/foldername/`
+- Check permission with `chmod 0755 TUN.sh`
+- Run it with `./TUN.sh`
+- Create the DNS config file by typing `vim /volume1/foldername/resolv.conf`
+- Paste the following lines :
+```
+nameserver 8.8.8.8
+nameserver 8.8.4.4
+```
+- Save the file with [escape] + `:wq!`
+- Create your docker container with `docker run --privileged -d -v /volume1/foldername/resolv.conf:/etc/resolv.conf -v /volume1/yourpath/:/data -e "OPENVPN_PROVIDER=PIA" -e "OPENVPN_CONFIG=Netherlands" -e "OPENVPN_USERNAME=XXXXX" -e "OPENVPN_PASSWORD=XXXXX" -p 9092:9091 haugene/transmission-openvpn -name TransmissionVPN`
+- If you wish to access WebUI, just run the nginx container as explained in doc on ports 9091:9091
```

Comment:
With the run command above, and nginx -p 9091:9091 as you say, they would have to change nginx.conf to:
* listen 9091;
* proxy_pass http://host.ip.address.here:9092;

Could be mentioned...